os/rtems/rtems-fio-wrap.h

```diff
@@ -0,0 +1,54 @@
+#ifdef __rtems__
```

Comment:
You already had that checked when you included that file. So you don't need it here.